### DISCUSSION TITLE
Pair Extraordinaire

### DISCUSSION THREAD

**Discussion by seldish-og** [ParanoidUser/yolo#149] [Galaxy Trivia] [answered]
Can you help me with Pair Extraordinaire? I heelp you too with anything I can. Also if you dont mind, can you like my most popular repository)

**Comment by kjbaker-uk** [comment, marked as answer]
Hey if you follow this guide you will get the achievement very quick. https://github.com/ParanoidUser/yolo/discussions/26

**Reply by seldish-og** [reply]
idk how I learn it. I didn't follow your guide but get this ach! Thank u anyway!

**Reply by kjbaker-uk** [reply]
You co-authored commits on merged pull requests.
